Bottom: be8a049c6be798337807c84d98c5aa4042b02910
Top:    8b2394eab45312f4f421983062769f05b862b863
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-28 00:47:52 +0100

Refresh of cure-cross-hair-lagging-on-scr

---

diff --git a/src/hid/gtk/gtkhid-gdk.c b/src/hid/gtk/gtkhid-gdk.c
index 52f299a..27d80f7 100644
--- a/src/hid/gtk/gtkhid-gdk.c
+++ b/src/hid/gtk/gtkhid-gdk.c
@@ -784,6 +784,7 @@ ghid_notify_crosshair_change (bool changes_complete)
     CrosshairOn ();
   else
     CrosshairOff ();
+  ghid_draw_area_update (gport, NULL);
 }
 
 static void
